parameters.cpp

```diff
@@ -441,7 +441,8 @@ bool P::addParameters() {
    RP::add("fieldtracing.tracer_max_allowed_error", "Maximum allowed error for the adaptive field line tracers ", 1000);
    RP::add("fieldtracing.tracer_max_attempts", "Maximum allowed attempts for the adaptive field line tracers", 100);
    RP::add("fieldtracing.tracer_min_dx", "Minimum allowed field line tracer step length for the adaptive field line tracers (m)", 100e3);
-   RP::add("fieldtracing.tracer_max_incomplete_fieldlines_fullbox", "Maximum fraction of field lines left incomplete when stopping tracing loop. Defaults to zero to process all, can be slow at scale!", 0);
+   RP::add("fieldtracing.fullbox_max_incomplete_fieldlines", "Maximum fraction of field lines left incomplete when stopping tracing loop. Defaults to zero to process all, will be slow at scale!", 0);
+   RP::add("fieldtracing.fluxrope_max_incomplete_fieldlines", "Maximum fraction of field lines left incomplete when stopping tracing loop. Defaults to zero to process all, will be slow at scale!", 0);
    RP::add("fieldtracing.use_reconstruction_cache", "Use the cache to store reconstruction coefficients. (0: don't, 1: use)", 0);
    RP::add("fieldtracing.fluxrope_max_curvature_radii_to_trace", "Maximum number of seedpoint curvature radii to trace forward and backward from each DCCRG cell to find flux ropes", 10);
```

Comment:
suggestion: "from each DCCRG cell to find flux ropes" -> "from each DCCRG cell when attempting to identify flux ropes"